src/include/disks.h

```diff
@@ -214,12 +214,27 @@ fatfs_t *get_fat_fs_by_drive(unsigned char drv_num);
 #define DERR_WP 	3
 #define DERR_NOTFOUND 	4
 #define DERR_CHANGE 	6
+#define DERR_BOUNDARY	9
 #define DERR_ECCERR 	0x10
 #define DERR_CONTROLLER	0x20
 #define DERR_SEEK	0x40
 #define DERR_NOTREADY	0x80
 #define DERR_WRITEFLT	0xcc
 
+/* Int13 maximum amount of sectors to access.
+ *
+ * Mentioned in RBIL 61 Int 1301 Table 00234,
+ *  which lists error code 09 as follows:
+ *
+ * 09h	data boundary error (attempted DMA across 64K boundary or >80h sectors)
+ *
+ * Note that > 80h sectors with 512 bytes per sector always crosses a 64 KiB
+ *  boundary, so it may be the error code is only meant for that crossing.
```

Comment:
IIRC DMA controller has the "page" register
where you specify the 64K-sized page number
for DMA window. Then you also specify the
address inside that window and transfer length.
I think this error means that the address+length
cross the 64K boundary, in which case DMA
will probably not work. I can see this code:
```
    void *addr = physaddr_to_unixaddr(
            (chan->page << 16) | (chan->cur_addr.value << dma_idx));
```
in `dma.c`. Of course I don't remember why I
wrote `|` instead of `+`, in which case the boundary
crossing could work (but maybe I consulted the
manual back then, so better not to touch).